client/src/legacy/entwine/LeftAndMain_Subsites.js

```diff
@@ -0,0 +1,190 @@
+/* jslint browser: true, nomen: true */
+/* global window, jQuery */
+import React from 'react';
+import ReactDom from 'react-dom';
+import { loadComponent } from 'lib/Injector';
+import createEvent from 'legacy/createEvent';
+import changeActiveSubsite from 'lib/changeActiveSubsite';
+
+jQuery.entwine('ss', ($) => {
```

Comment:
`$ =>` instead of `($) =>` - there's multiple bits where this can be updated

Reply:
As with https://github.com/silverstripe/silverstripe-subsites/pull/443#discussion_r384174797 I'll not make a change until this discussion reaches an outcome.